### ISSUE TITLE
Add INTERFACE_DOWN for ios

### ISSUE THREAD

**Issue by nvinzens** [napalm-automation/napalm-logs#196]
Since there is are no profiles for ios currently I started to work on the INTERFACE_DOWN profile for which I have a use case.
I wrote init.yml and INTERFACE_DOWN.yml and placed them in the new napalm_logs/config/ios. After starting napalm-logs I could see that it started a worker for ios in the log-file. For some reason the regex doesn't match and I can't seem to figure out why.
So I thought I'd post what I have so far here:
init.yml
```
#<189>521: router1: *Nov 14 08:30:56.699: %LINK-5-CHANGED: Interface GigabitEthernet2, changed state to administratively down

prefixes:
  - time_format: "%b %d %H:%M:%S"
    values:
      pri: (\d+)
      messageId: (\d+)
      host: (\w+\d)
      date: (\w+ \d+)
      time: (\d\d:\d\d:\d\d.\d\d\d)
      tag: ([\w\d-]+)
      message: (.*)
    line: '\<{pri}\>{messageId}: {host}: \*{date} {time}: %{tag}:'
```
INTERFACE_DOWN.yml
```messages:
  - error: INTERFACE_DOWN
    tag: LINK-5-CHANGED
    values:
      interface: (\w+[\.\-\d\/\w+]+)
      change_mode: (\w+)
      status: (\w+)
    line: 'Interface {interface}, changed state to {change_mode} {status}'
    model: openconfig-interfaces
    mapping:
      variables:
        interfaces//interface//{interface}//state//oper_status: status
      static: {}
```
The variables {change_mode} and {status} I'm not married to, I just thought they might prove useful in my particular case.
Once this profile works I'm probably going to work on adding other ios messages too (BGP probably)

**Comment by loverend** [comment]
`pri` and `message` are matched automatically so do not need to be defined. I will add this to the docs.

Also you cannot use special characters in the `values` keys as this causes issues with the regex.

Try:

```
prefixes:
  - time_format: "%b %d %H:%M:%S"
    values:
      messageId: (\d+)
      host: (\w+\d)
      date: (\w+ \d+)
      time: (\d\d:\d\d:\d\d)
      milliseconds: (\.\d\d\d)
      tag: ([\w\d-]+)
    line: '{messageId}: {host}: *{date} {time}{milliseconds}: %{tag}:'
```

```
messages:
  - error: INTERFACE_DOWN
    tag: LINK-5-CHANGED
    values:
      interface: (\w+[\.\-\d\/\w+]+)
      changeMode: (\w+)
      status: (\w+)
    line: 'Interface {interface}, changed state to {changeMode} {status}'
    model: openconfig-interfaces
    mapping:
      variables:
        interfaces//interface//{interface}//state//oper_status: status
      static: {}
```
This will produce:

```
{'yang_message': {'interfaces': {'interface': {'GigabitEthernet2': {'state': {'oper_status': 'down'}}}}}, 'message_details': {'severity': 5, 'facility': 23, 'messageId': '521', 'pri': '189', 'host': 'router1', 'tag': 'LINK-5-CHANGED', 'time': '08:30:56', 'date': 'Nov 14', 'message': 'Interface GigabitEthernet2, changed state to administratively down', 'milliseconds': '.699'}, 'facility': 23, 'ip': '127.0.0.1', 'error': 'INTERFACE_DOWN', 'host': 'router1', 'yang_model': 'openconfig-interfaces', 'timestamp': 1510648256, 'os': 'ios', 'severity': 5}
```

You will notice that I have separated `time` into `time` and `milliseconds`, this is because the time needs to match `time_format`.

**Comment by nvinzens** [comment]
Works great, thanks!

**Comment by mirceaulinic** [comment]
@nvinzens if this works fine, would send a PR to add this IOS profile? The community asked often about this one, so it would be appreciated!

**Comment by nvinzens** [comment]
Sure, this would be my first PR though, so I'm not completely sure how it works. I also think that not everything is like the community would like it to be (eg. haven't written any tests, like I saw in another PR). Can I just create a PR and then let the community comment on it / change it?

**Comment by mirceaulinic** [comment]
Sure. just submit a PR, and then we'll guide you if there's anything else to adjust / correct / improve. Thanks!

**Comment by mirceaulinic** [comment]
This has been solved via #218. Looking forward to more @nvinzens. Thanks! :-)
